t/t-multiple-remotes.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+# Test lfs capability to download data when blobs are stored in different
+# endpoints
+
+. "$(dirname "$0")/testlib.sh"
+
+reponame="$(basename "$0" ".sh")"
+
+begin_mrtest(){
+  # The global configuration sets a common http endpoint (ip:port) for lfs
+  # We deactivate the git global configuration related with the http protocol
+  # by setting a new .gitconfig file
+  rm -rf "$HOME"/.gitconfig
+  git config --global user.name "Git LFS Tests"
+  git config --global user.email "git-lfs@example.com"
+}
+
+prepare_consumer() {
+  local consumer="$1"
+  mkdir "$consumer"
+  cd "$consumer"
+  git init
+  git remote add mr file://"$smain"
+  git remote add fr file://"$sfork"
+  git fetch mr
+  git fetch fr
+  git lfs install
+}
+
+prepare_remote() {
+  #Set up remote repo that does not use the http API
+  local repodir="$1"
+  mkdir -p "$repodir"
+  cd "$repodir"
+  git init --bare
+  #Uses the basic stand alone transfer instead of the http API
+  git lfs install
+}
+
+prepare_forks () {
+  local testcase="$1"
+  smain="$HOME"/"$reponame"-"$testcase"-main-remote.git
+  sfork="$HOME"/"$reponame"-"$testcase"-fork-remote.git
+  cmain="$HOME"/"$reponame"-"$testcase"-main-repo
+  cfork="$HOME"/"$reponame"-"$testcase"-fork-repo
+  prepare_remote "$smain"
```

Comment:
Can we change this to `setup_remote_repo "$smain"` and then use `"$REMOTEDIR/$smain.git"`?  We shouldn't need the `git lfs install` since that's already been done once by our setup code, and if we don't delete the `.gitconfig` file as I suggested, then we'll have the configuration persisted.

Basically, my request here is to use as much of the same setup code and other helper functions as possible.  The style of this test is currently very different from our other tests, and if we can make it as similar as possible, then it will be easier to maintain.  You can see what `t/t-verify.sh` looks like, which I think is a good example of a typical test.

Reply:
You are right, I adpted the tests to use the function `setup_remote_repo()`